.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-ruby@v2
+      with:
+        ruby-version: '2.7'
+
+    - uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: bundle-${{ hashFiles('Gemfile.lock') }}
+
+    - name: Install Dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get -yqq install libsqlite3-dev chromium-browser
+        bundle install --jobs 4 --retry 3
```

Comment:
I've cribbed this GitHub Action configuration file from other Rails engines' suites. Unfortunately, this hotwired/stimulus-rails doesn't bundle `bin/rails` (only `bin/test`), so I'm not sure how to install the Selenium server through [webdrivers](https://github.com/titusfortner/webdrivers).

@javan I might need some administrative assistance in getting the GitHub actions set up to serve as CI.